sdk/keyvault/keyvault-keys/src/keyVaultCryptographyClient.ts

```diff
@@ -0,0 +1,690 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
```

Comment:
If it makes reviewing easier this was basically a rename, everything in this file is unchanged as far as functionality (this used to be the `CryptographyClient` class when we had both this and local crypto client exposed

Reply:
As such, I'm only taking a cursory glance at this file.